### DISCUSSION TITLE
算法标题的计数器调整

### DISCUSSION THREAD

**Discussion by martin4396** [tuna/thuthesis#665] [Q&A] [answered]
作者您好，感谢您的付出。我是正在做毕设的本科生，正在用algorithm和algorithmic包在写算法。我现在希望算法编号能够呈现2.1和2.2，而非直接按照2和3编码下去，请问应该怎么实现？

![捕获](https://user-images.githubusercontent.com/55222414/119254966-c151bc00-bbeb-11eb-814f-dfd86b95d1e7.jpg)

**Comment by zepinglee** [comment, marked as answer]
附录中的章号切换成 A, B, C，算法编号跟图表一样，也是切换成 A.1, A.2 的形式。

**Comment by Harry-Chen** [comment]
不推荐手工编号，而应该使用 `\ref`
